src/service/document-info-impl.js

```diff
@@ -104,7 +105,15 @@ export class DocInfo {
       },
       canonicalUrl,
       pageViewId,
-      pageViewId64,
+      get pageViewId64() {
```

Comment:
Does this mean we are relying on ` get pageViewId64()` getting called after extensions service is ready?

Reply:
Only after it's registered which happens very soon afterwards in `runtime.adopt()`.